scripts/obtain/dotnet-install.sh

```diff
@@ -84,6 +84,10 @@ get_current_os_name() {
                     echo "opensuse.13.2"
                     return 0
                     ;;
+                "opensuse.42.1")
```

Comment:
Could you clarify this part? Are we replacing this whole file or something? This was just a part of the pre-existing initialization process. Don't the CLI changes need to follow after this stuff is already merged and built?